flake.nix

```diff
@@ -47,6 +48,8 @@
           "${inputs.plutarch-quickcheck}"
           "${inputs.plutarch-numeric}"
           "${inputs.plutarch-context-builder}"
+          "${inputs.ply}/ply-core"
```

Comment:
I don't know if it's advisable, but I used `${inputs.liqwid-script-export.inputs.ply}/ply-{core|plutarch}` for other repositories.

Reply:
We have a cyclic dependency issue here.

Reply:
yeah, oh no. Perhaps we just put all ply orphans into LSE?

Reply:
We have very confusing orphan situation:

Agora and LPE both have orphans for Aeson instances. Propbably liqwid also has its own.
LSE requires LPE for all Aeson instances and Ply instances (if we decide to put it on LPE).

Since now, we don't need Aeson instances for individual onchain repo, we can put all Aeson instances and Ply instances into LSE. I think that will make LSE independent of LPE.

Reply:
That makes sense. Let's just go with this. :+1: